doc/developers-guide/domain_management.md

```diff
@@ -129,9 +129,8 @@ curl -v -X PUT "http://localhost:8088/api/domains/example.db" \
 Result codes:
 
 * 204 - inserted.
+* 403 - DB service disabled, or an unknown host type.
```

Comment:
Looks like https://github.com/esl/MongooseIM/blob/master/doc/rest-api/Dynamic-domains_swagger.yml#L53 text should be without comma and should be used as `DB service disabled or the host type is unknown.`. I suppose need compare current all text and in Swagger.